.github/workflows/calibreapp-image-actions.yml

```diff
@@ -0,0 +1,12 @@
+name: Compress images
```

Comment:
Can you capitalize 'images' here so it matches the naming style of the other workflow?